commands/command_locks.go

```diff
@@ -19,7 +19,12 @@ var (
 )
 
 func locksCommand(cmd *cobra.Command, args []string) {
-	filters, err := locksCmdFlags.Filters()
+	lockData, err := computeLockData()
+	if err != nil {
+		Error(err.Error())
+		os.Exit(3)
```

Comment:
My comments above about `ExitWithError()` apply here too, obviously enough.

I like that we're running `computeLockData()` here before `newLockClient()` so we don't have to close the client on error.